FiniteCocompletions/tst/100_LoadPackage.tst

```diff
@@ -15,6 +15,8 @@ gap> LoadPackage( "FunctorCategories", false );
 true
 gap> LoadPackage( "LazyCategories", false );
 true
+gap> LoadPackage( "Locales", false );
+true
```

Comment:
The aim of making ComplexesCategories a suggested dependency was to avoid having to load Locales everywhere. So instead of loading Locales here, it should be possible to simply revert the changes in https://github.com/homalg-project/CategoricalTowers/pull/266.

Reply:
I loaded `Locales` to avoid touching the tests. It is now @mohamed-barakat 's call.

Reply:
Loading `Locales` in the tests should be ok.

Reply:
> Loading `Locales` in the tests should be ok.

Let's discuss this verbally.

Reply:
> Let's discuss this verbally.

As discussed, I will merge, then remove the `Locales` dependency in the tests of `Toposes` later.